play/src/front/Chat/Connection/Matrix/MatrixRoomFolder.ts

```diff
@@ -267,6 +300,10 @@ export class MatrixRoomFolder extends MatrixChatRoom implements RoomFolder {
                 console.error("Failed to refresh suggested rooms:", error);
                 Sentry.captureException(error);
             });
+            this.refreshAllChildRooms().catch((error) => {
+                console.error("Failed to refresh all child rooms:", error);
```

Comment:
Same thing, here, could we imagine having information for the user and/or button to try again?